test/travis.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# Wrapper for travis tests
+
+function docker_build {
+  docker build -t httpse .
+}
+
+# Folder paths, relative to parent
+RULESETFOLDER="src/chrome/content/rules"
+
+# Go to git repo root; taken from ../test.sh. Note that
+# $GIT_DIR is .git in this case.
+if [ -n "$GIT_DIR" ]
+then
+  # $GIT_DIR is set, so we're running as a hook.
+  cd $GIT_DIR
+  cd ..
+else
+  # Git command exists? Cool, let's CD to the right place.
+  git rev-parse && cd "$(git rev-parse --show-toplevel)"
+fi
+
+# Fetch the current GitHub version of HTTPS-E to compare to its master
+git remote add upstream-for-travis https://github.com/EFForg/https-everywhere.git
+trap 'git remote remove upstream-for-travis' EXIT
```

Comment:
This was helping when writing the PR.  I wouldn't say that this will *always* be run by travis, though.  There may be an instance when we're double-checking the travis build (for instance, we did this when we discovered lack of travis ipv6 support), and there are no negative consequences from keeping it in (other than increasing the build time by a fraction of a second).